test/1_stdlib/UIKit.swift

```diff
@@ -1,77 +1,77 @@
+  return s
+}
+
+private func printInterface(o: UIInterfaceOrientation) -> String {
+  return "\(o.isPortrait) \(UIInterfaceOrientationIsPortrait(o)), " +
+    "\(o.isLandscape) \(UIInterfaceOrientationIsLandscape(o))"
+}
+
+UIKitTests.test("UIDeviceOrientation") {
+  expectEqual("false false, false false, false, false false",
+    printDevice(.unknown))
+
+  expectEqual("true true, false false, false, true true",
+    printDevice(.portrait))
+
+  expectEqual("true true, false false, false, true true",
+    printDevice(.portraitUpsideDown))
+
+  expectEqual("false false, true true, false, true true",
+    printDevice(.landscapeLeft))
+
+  expectEqual("false false, true true, false, true true",
+    printDevice(.landscapeRight))
+
+  expectEqual("false false, false false, true, false false",
+    printDevice(.faceUp))
+
+  expectEqual("false false, false false, true, false false",
+    printDevice(.faceDown))
+}
+
+UIKitTests.test("UIInterfaceOrientation") {
+  expectEqual("false false, false false",
+    printInterface(.unknown))
+
+  expectEqual("true true, false false",
+    printInterface(.portrait))
+
+  expectEqual("true true, false false",
+    printInterface(.portraitUpsideDown))
+
+  expectEqual("false false, true true",
+    printInterface(.landscapeLeft))
+
+  expectEqual("false false, true true",
+    printInterface(.landscapeRight))
+}
+
+UIKitTests.test("UIEdgeInsets") {
+  let insets = [ UIEdgeInsets(top: 1.0, left: 2.0, bottom: 3.0, right: 4.0),
+    UIEdgeInsets(top: 1.0, left: 2.0, bottom: 3.1, right: 4.0), UIEdgeInsets.zero ]
+  checkEquatable(insets, oracle: { $0 == $1 })
+  expectEqual(UIEdgeInsets.zero, UIEdgeInsets.zero)
 }
 
-print("Device orientations")
-printOrientation(UIDeviceOrientation.unknown)
-printOrientation(UIDeviceOrientation.portrait)
-printOrientation(UIDeviceOrientation.portraitUpsideDown)
-printOrientation(UIDeviceOrientation.landscapeLeft)
-printOrientation(UIDeviceOrientation.landscapeRight)
-printOrientation(UIDeviceOrientation.faceUp)
-printOrientation(UIDeviceOrientation.faceDown)
-// CHECK:      Device orientations
-// CHECK-NEXT: false false, false false, false, false false
-// CHECK-NEXT: true true, false false, false, true true
-// CHECK-NEXT: true true, false false, false, true true
-// CHECK-NEXT: false false, true true, false, true true
-// CHECK-NEXT: false false, true true, false, true true
-// CHECK-NEXT: false false, false false, true, false false
-// CHECK-NEXT: false false, false false, true, false false
-
-
-func printOrientation(o: UIInterfaceOrientation) {
-  print("\(o.isPortrait) \(UIInterfaceOrientationIsPortrait(o)), ", terminator: "")
-  print("\(o.isLandscape) \(UIInterfaceOrientationIsLandscape(o))")
+UIKitTests.test("UIOffset") {
+  let offsets = [ UIOffset(horizontal: 1.0, vertical: 2.0),
+    UIOffset(horizontal: 1.0, vertical: 3.0), UIOffset.zero ]
+  checkEquatable(offsets, oracle: { $0 == $1 })
+  expectEqual(UIOffset.zero, UIOffset.zero)
```

Comment:
You shouldn't need that last `expectEqual()` check here (and in the test above).  `checkEquatable` should be doing it.